pages/linux/ksvgtopng5.md

```diff
@@ -0,0 +1,7 @@
+# ksvgtopng5
+
+> Convert SVG files to PNG format.
+
+- Convert an SVG file (should be an absolute path) to PNG:
+
+`ksvgtopng5 {{width}} {{height}} {{full_path/to/file.svg}} {{output_filename.png}}`
```

Comment:
I think we can mention the bit about full path at only one place. Either the description or the example. I am fine with either.

Reply:
I put it in the description

Reply:
Sure, can we then make the file path as "path/to/file.svg"?

Reply:
done